run.sh

```diff
@@ -90,7 +92,7 @@ echo "this shouldn't take more than a minute.."
 num=0
 while true; do
   num=$(wget --quiet -O - "http://$graphitemon_host:8000/render/?target=graphite-watcher.$env.num_metrics&from=-2min&until=-10s&format=raw" | sed -e 's#.*,##')
-  [ "$num" == "${total}.0" ] && break
+  (( $(echo  "$num >=  $total"|bc -l) )) && break
```

Comment:
why do we need a subshell and 2 external processes to compare two numbers? what was wrong with the original code?